src/mcts/search.cc

```diff
@@ -1515,14 +1515,19 @@ int SearchWorker::WaitForTasks() {
   }
 }
 
+void SearchWorker::ProactivelyNotifyTaskWorkers() {
+  if (task_workers_ <= 0 || search_->network_->IsCpu())
+    return;
+
+  // While nothing is ready yet - wake the task runners so they are ready to
+  // receive quickly.
+  Mutex::Lock lock(picking_tasks_mutex_);
+  task_added_.notify_all();
+}
+
 void SearchWorker::PickNodesToExtend(int collision_limit) {
   ResetTasks();
-  {
```

Comment:
Do we need a separate method for this? Why not just change this line to
```c++
if (task_workers_ > 0 && !search_->network_->IsCpu()) {
```

Reply:
Right, done, thanks!